feature/tabsflow/moreflow/donate/src/main/java/com/epmedu/animeal/tabs/more/donate/presentation/DonateListItem.kt

```diff
@@ -0,0 +1,78 @@
+package com.epmedu.animeal.tabs.more.donate.presentation
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealIcon
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.foundation.theme.CustomColor.secondaryBackground
+import com.epmedu.animeal.resources.R
+
+@Composable
+internal fun DonateListItem(
+    modifier: Modifier = Modifier,
+    @DrawableRes icon: Int,
+    header: String,
+    bankNumber: String,
+    onEvent: (event: DonateScreenEvent) -> Unit,
+) {
+    Column(modifier = modifier) {
+        Text(
```

Comment:
In your preview it looks to close the header and the donation info, may be add some spacer or padding to the text. What do you think?